TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Security/Authorization/Privilege/Node/CreateNodePrivilege.php

```diff
@@ -38,34 +38,53 @@ class CreateNodePrivilege extends AbstractNodePrivilege
      */
     public function matchesSubject(PrivilegeSubjectInterface $subject)
     {
-        if ($subject instanceof CreateNodePrivilegeSubject === false && $subject instanceof MethodPrivilegeSubject === false) {
-            throw new InvalidPrivilegeTypeException(sprintf('Privileges of type "%s" only support subjects of type "%s" or "%s", but we got a subject of type: "%s".', CreateNodePrivilege::class, CreateNodePrivilegeSubject::class, MethodPrivilegeSubject::class, get_class($subject)), 1417014353);
+        if (!($subject instanceof CreateNodePrivilegeSubject) && !($subject instanceof MethodPrivilegeSubject)) {
```

Comment:
The extra brackets are not needed, `if(!$subject instanceof ...)` works just find. But I'm not sure if that's better than the previous format tbh